src/Runner/Runner.php

```diff
@@ -95,7 +90,7 @@ public function run()
 			$handler->begin();
```

Comment:
I'd love to be able to get `$this->jobCount` in here, but that would mean moving this call after `$this->findTests()` in which results may already be sent to output. They'd have to be buffered somehow. What do you think?